gfx/webrender/src/bindings.rs

```diff
@@ -286,7 +286,7 @@ pub extern fn wr_create(width: u32, height: u32, counter: u32) -> *mut WrState {
     // let notifier = Box::new(Notifier::new(window.create_window_proxy()));
     // renderer.set_render_notifier(notifier);
 
-    let pipeline_id = PipelineId(0, counter);
+    let pipeline_id = PipelineId((layers_id >> 32) as u32, layers_id as u32);
```

Comment:
any reason you are not storing and passing `layers_id` as `PipelineId` right away? Seems to be converted like this in a few places.